packages/components/molecules/f-restaurant-card/src/components/subcomponents/IconText.vue

```diff
@@ -67,16 +67,17 @@ export default {
 <style lang="scss" module>
 @mixin themedColor($color) {
     color: $color;
+
     path {
         fill: $color;
     }
 }
 
 .c-restaurantCard-iconText {
-    @include font-size();
+    @include font-size($font-body-s-paragraph);
```

Comment:
```suggestion
    @include font-size(body-s);
```

Should give the same result.